### PR TITLE
Updated states (provinces) in do.xml according to ISO 3166-2.

### PR DESCRIPTION
**Pull request by rickygzz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please read below.
| Fixed ticket? | Fixes #14 (part of it, more PR's are coming :)

Added \<states> according to ISO 3166-2 (https://www.iso.org/obp/ui/#iso:code:3166:DO).

### CODE REVIEW

File: do.xml [hunk 1]
```diff
@@ -18,6 +18,49 @@
...
 	</taxes>
+	<states>
+		<state name="Azua" iso_code="02" country="DO" zone="Central America/Antilla" />
+		<state name="Baoruco" iso_code="03" country="DO" zone="Central America/Antilla" />
+		<state name="Barahona" iso_code="04" country="DO" zone="Central America/Antilla" />
+		<state name="Cibao Nordeste" iso_code="33" country="DO" zone="Central America/Antilla" />
+		<state name="Cibao Noroeste" iso_code="34" country="DO" zone="Central America/Antilla" />
+		<state name="Cibao Sur" iso_code="36" country="DO" zone="Central America/Antilla" />
+		<state name="Dajabón" iso_code="05" country="DO" zone="Central America/Antilla" />
+		<state name="Distrito Nacional (Santo Domingo)" iso_code="01" country="DO" zone="Central America/Antilla" />
```
Comment (Julievrz) [inline do.xml]: Isn't Cibao Norte missing?

Comment (Julievrz) [inline do.xml]: Distrito Nacional (Santo Domingo)

It's more than 32 characters. Could we keep Santo Domingo only?

File: do.xml [hunk 2]
```diff
@@ -18,6 +18,49 @@
...
+		<state name="San Cristóbal" iso_code="21" country="DO" zone="Central America/Antilla" />
+		<state name="San José de Ocoa" iso_code="31" country="DO" zone="Central America/Antilla" />
+		<state name="San Juan" iso_code="22" country="DO" zone="Central America/Antilla" />
+		<state name="San Pedro de Macorís" iso_code="23" country="DO" zone="Central America/Antilla" />
+		<state name="Sánchez Ramírez" iso_code="24" country="DO" zone="Central America/Antilla" />
+		<state name="Santiago" iso_code="25" country="DO" zone="Central America/Antilla" />
+		<state name="Santiago Rodríguez" iso_code="26" country="DO" zone="Central America/Antilla" />
+		<state name="Santo Domingo" iso_code="32" country="DO" zone="Central America/Antilla" />
+		<state name="Valdesia" iso_code="41" country="DO" zone="Central America/Antilla" />
+		<state name="Valverde Province" iso_code="27" country="DO" zone="Central America/Antilla" />
```
Comment (Julievrz) [inline do.xml]:
```suggestion
		<state name="Valverde" iso_code="27" country="DO" zone="Central America/Antilla" />
```

It was written "Valverde" only on https://www.iso.org/obp/ui/#iso:code:3166:DO

**Comment by Progi1984** [comment]
Ping @rickygzz

**Review by kpodemski — APPROVED** [review]

**Pull request merged**